### DISCUSSION TITLE
Your Software

### DISCUSSION THREAD

**Discussion by Starsurfer78** [acredsfan/autonomous_mower#2] [General]
Hi,
I run a website for Diy Robot Lawn Mower. 
There, we recently came across the topic of PiMowBot. At the moment, we use a Teensy 4.1 PCB and ESP32 for our Mowers.
But I would also like to build a mower with Raspberry. Unfortunately, the main code of the PiMowBot is closed source, which is why I came across your project.
Maybe you would like to get in touch with me and/or present your project on https://www.diy-robot-lawn-mower.com/?

**Comment by acredsfan** [comment]
> Hi,
> I run a website for Diy Robot Lawn Mower. 
> There, we recently came across the topic of PiMowBot. At the moment, we use a Teensy 4.1 PCB and ESP32 for our Mowers.
> But I would also like to build a mower with Raspberry. Unfortunately, the main code of the PiMowBot is closed source, which is why I came across your project.
> Maybe you would like to get in touch with me and/or present your project on https://www.diy-robot-lawn-mower.com/?

Hi,

I am definitely happy to connect/let you share the project.  As you've probably noticed it's still very much a work in progress.  I try to work on it as I get free time but it's very sporadic and I welcome any input from others who would like to contribute.

**Comment by Starsurfer78** [comment]
As I said, it would be great if you could register with us in the forum and present your project.
We are a small community, but there are certainly people who can contribute ideas.
Does the software work in its current state, so can you already work with it?
Since we rebuild defective robotic mowers, we have other requirements, e.g. 29V and charging station, etc.

**Reply by acredsfan** [reply]
I just registered and posted my project on your forum.  The software is not yet completely working but most of the individual components are working.  I've begun focusing on the path_planning module and main.py to pull everything together.
